packages/core/src/sequelize.js

```diff
@@ -743,25 +743,23 @@ Use Sequelize#query if you wish to use replacements.`);
    * Execute a query which would set an environment or user variable. The variables are set per connection, so this function needs a transaction.
    * Only works for MySQL or MariaDB.
    *
-   * @param {object}        variables Object with multiple variables.
-   * @param {object}        [options] query options.
-   * @param {Transaction}   [options.transaction] The transaction that the query should be executed under
-   *
-   * @memberof Sequelize
+   * @param {object} variables Object with multiple variables.
+   * @param {object} [options] query options.
    *
    * @returns {Promise}
    */
-  async set(variables, options) {
```

Comment:
Don't we want to deprecate `sequelize.set` first (aka `set` is an alias for `setSessionVariables`) and remove it later? Or is it used too little to need it?

Reply:
Considering we're also renaming the global option, I think it's safer to not deprecate it and document the breaking change instead